SingularityService/src/main/java/com/hubspot/singularity/resources/TaskResource.java

```diff
@@ -180,6 +183,19 @@ public SingularityTaskRequest getPendingTask(@Auth SingularityUser user, @PathPa
     return taskRequestList.get(0);
   }
 
+  @DELETE
+  @Path("/scheduled/task/{scheduledTaskId}")
+  @ApiOperation("Delete a scheduled task.")
+  public void deleteScheduledTask(@Auth SingularityUser user, @PathParam("scheduledTaskId") String taskId) {
+    SingularityTaskRequest taskRequest = getPendingTask(user, taskId);
+    SingularityRequest request = taskRequest.getRequest();
+
+    authorizationHelper.checkForAuthorization(request, user, SingularityAuthorizationScope.ADMIN);
```

Comment:
this will need to be check by request and a scope of WRITE. The user just needs to be authorized for the request they are operating on, not necessarily for admin actions